config-templates/ibm/ibm-vpc-block-csi-driver/5.0/storage-class-parameters.json

```diff
@@ -53,6 +65,9 @@
         "required": "false",
         "default": "",
```

Comment:
In our opensource, we have kept a default IOPS of 400, should be this kept empty, I am not sure, @sameshai please confirm

Reply:
yes iops is only possible for custom storage class lets set default value to 400.

Reply:
please set the default value to 400

Reply:
Addressed in [3ffae75](https://github.com/IBM/ibm-satellite-storage/pull/271/commits/3ffae750f874ad7ae3fd4c11be92e11ea2443e03) build